StringConcatenate.txt

```diff
@@ -0,0 +1,9 @@
+/*
```

Comment:
Commit messages are not in format, please check it.
The PR title is supposed to be capitalised.

Reply:
I will change the title and will check commit messages' format.